docs/ELRS.md

```diff
@@ -0,0 +1,62 @@
+# mLRS Documentation: ExpressLRS Devices #
+
+([back to main page](../README.md))
+
+In principle, any ELRS receiver which uses the ESP8285 chipset is supported. This page does not list all of them, but just some which are considered particularly usefull for being used as mLRS receiver, e.g., because their transmit power is >= 100 mW. 
+
+ELRS receivers using a EPS32 chipset or ELRS transmitter modules are currently not (yet) supported.
+
+## Receivers ##
+
+### SpeedyBee Nano 2.4G ExpressLRS ELRS Receiver
```

Comment:
Am wondering if it makes more sense to do this by ELRS target then include all of the hardware that has that target.  I could take up this in subsequent PR.

Reply:
that's a good point for discussion

the device_config.h and hal.h split allows to use generic hals for several boards
so, we can follow two strategies, we could 
- (1) provide precompiled firmwares for the generic targets and somewhere dosument which board needs to be flashed with which, or 
- (2) provide preccomiled firmwares for the specific boards, with maybe some documentation somewhere what to use in case of not specific boards

currently I'm thinking more in terms of (2), but you guys really can overrule me